ls1intum/Artemis
# `Text exercises`: Fix breadcrumbs during assessment

### Checklist
#### General


#### Server
#### Client

### Motivation and Context


Fix #6778

### Description

The text exercise was the only one that used the participationId in the browser URL as well. This also affected the breadcrumbs that could not handle that "participations" were followed by "submissions".
This PR unifies the browser URL of text exercises as well as the REST URL of `retrieveParticipationForSubmission` with the other exercise types to not use the participation id.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Text Exercise

1. Log in to Artemis
2. Participate in the text exercise as student
3. Assess the exercise as tutor and check that everything (starting, continuing, submitting the assessment, ...) still works.
4. Check that the breadcrumbs during the assessment work as expected. "submissions" should redirect to the scores page.

#### Exam Mode Testing



Prerequisites:
- 1 Instructor
- 1 Students
- 1 Text Exercise

1. Log in to Artemis
2. Participate in the text exercise as student
3. Assess the exercise as tutor and check that everything (starting, continuing, submitting the assessment, ...) still works.
4. Check that the breadcrumbs during the assessment work as expected. "submissions" should redirect to the scores page.

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage
